packages/strapi-design-system/src/Popover/Popover.tsx

```diff
@@ -24,7 +24,6 @@ export const POPOVER_PLACEMENTS = [
 
 const PopoverWrapper = styled(Box)`
   box-shadow: ${({ theme }) => theme.shadows.filterShadow};
-  z-index: 4;
```

Comment:
isn't this going to fail if the popover is in a modal? 🤔

Reply:
Also would it not be better to use the `zIndexes` in the theme? & update the header the same way but with a higher theme value? :)